### PR TITLE
Set up a vanity URL for build02's console

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-1432

https://coreos.slack.com/archives/C6A3NV5J9/p1592833393137200

```
oc --context build02 get console.operator cluster -o yaml
```

/cc @alvaroaleman @jhadvig

### CODE REVIEW

File: clusters/build-clusters/02_cluster/openshift-console/cluster_console.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: config.openshift.io/v1
```
Comment (spadgett) [inline clusters/build-clusters/02_cluster/openshift-console/cluster_console.yaml:1]:
```suggestion
apiVersion: operator.openshift.io/v1
```

File: clusters/build-clusters/02_cluster/openshift-console/cluster_console.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+kind: Console
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  route:
+    hostname: console.build02.ci.openshift.org
+    secret: console-build02-tls
```
Comment (spadgett) [inline clusters/build-clusters/02_cluster/openshift-console/cluster_console.yaml:10]:
```suggestion
    secret:
      name: console-build02-tls
```

File: clusters/build-clusters/02_cluster/openshift-console/cluster_console.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  managementState: Managed
```
Comment (hongkailiu) [inline clusters/build-clusters/02_cluster/openshift-console/cluster_console.yaml:8]: This is taken from the existing one.

**Comment by spadgett** [comment]
Adding a hold in case @alvaroaleman wants to review the other changes in the PR

**Pull request merged**